app/src/main/java/org/jellyfin/androidtv/util/profile/ExoPlayerProfile.kt

```diff
@@ -195,16 +196,25 @@ class ExoPlayerProfile(
 				)
 			})
 			// HEVC profiles
-			add(deviceHevcCodecProfile)
-			addAll(deviceHevcLevelCodecProfiles)
+			if (!DeviceUtils.has4kVideoSupport())   {
```

Comment:
What's the purpose of this? You're saying that if 4K isn't supported then just accept every profile no matter the level supported as long as the resolution is 1080p? How does that work?

This strikes me as very wrong.

Reply:
It tries to solve the problem of the player failing to play 4k HEVC content on 1080p devices. 4k content gets remuxed causing the player to crash. 

But it bypasses all checks, so yeah, this will not work. I haven't been able to find a workaround. I'll open it as an issue to provide more details, but I can't proceed with this until that gets resolved.